game/src/edit/roads.rs

```diff
@@ -508,8 +508,34 @@ fn can_reverse(_: LaneType) -> bool {
     lt == LaneType::Driving || lt == LaneType::Biking || lt == LaneType::Bus
 }*/
 
+fn default_outside_lane_placement(road: &mut EditRoad, dir: Direction) -> usize {
+    let idx = if road.lanes_ltr.first().unwrap().dir == dir {
+        if road.lanes_ltr.first().unwrap().lt.is_walkable() {
+            1
```

Comment:
Logic: Just talking myself through things here: what about the case of a one-way when we have

sdds
v^^^

(backwards sidewalk, fwds driving, fwds driving, fwds sidewalk)

That would insert a backwards driving lane at idx 1. Which is still the reasonable spot, so cool.